integrations/Amplitude/browser.js

```diff
@@ -338,22 +338,24 @@ class Amplitude {
     this.setDeviceId(rudderElement);
 
     const { properties, name, category } = rudderElement.message;
-
+    const { alterPageEventName } = rudderElement.message.context;
     // all pages
     if (this.trackAllPages) {
       const event = "Loaded a page";
       amplitude.getInstance().logEvent(event, properties);
     }
-
+    let event;
     // categorized pages
     if (category && this.trackCategorizedPages) {
-      const event = `Viewed page ${category}`;
+      if (!alterPageEventName) event = `Viewed page ${category}`;
+      else event = `Viewed ${category} Page`;
       amplitude.getInstance().logEvent(event, properties);
     }
 
     // named pages
     if (name && this.trackNamedPages) {
-      const event = `Viewed page ${name}`;
+      if (!alterPageEventName) event = `Viewed page ${name}`;
+      else event = `Viewed ${name} Page`;
       amplitude.getInstance().logEvent(event, properties);
```

Comment:
Minor re-factor
- localize `event` variable to the `if` blocks
- To improve readability 
`let event = "Viewed Page Category";
    if (<flag>) event = "Viewed Category Page"`